src/components/__test__/About.test.js

```diff
@@ -0,0 +1,7 @@
+import renderer from "react-test-renderer";
+import About from "../../pages/About/About";
```

Comment:
- You can use the `@` instead of the relative path `..` code will be like this.
`import About from "@/pages/About/About";`